app/models/photo.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class Photo < ApplicationRecord
+  belongs_to :reply
+  has_many_attached :images
```

Comment:
Wait, a photo record has many images? 🤯 Does Telegram provide different resolutions or why’s that?

Otherwise, if that’s just all the photos a user sent with a single message, why not name the record either `ReplyPhotos` or alternatively, attach the images directly to a `Reply`?

Reply:
Each photo comes with a somehow unpredictable set of resolutions. With my test photos yesterday, it was usually between 1-3. The last of them seems to be the original or highest resolution. Then you can send multiple photos as an album. The telegram API will send each photo with its image files in one message. It took me a while to get through that.

So the connection is Reply <1:m> Photo <1:m> attached_images.

Reply:
There are many use cases for a photo collection e.g. a document with more than one page.